src/parallax/fourth/button.module.scss

```diff
@@ -0,0 +1,12 @@
+.btn {
+  color: white;
+  text-transform: uppercase;
+  text-align: center;
+  background-color: orangered;
+  border-radius: 50px;
+  // Q: fixed width for button? 🤔
+  width: 200px;
```

Comment:
It's okay to have a width, as long you don't have a height. Then if the text is too large, it can break to the second line.